packages/browser-repl/src/lib/autocompleter/shell-api-autocompleter.ts

```diff
@@ -0,0 +1,30 @@
+import cliReplCompleter from 'mongosh-cli-repl/lib/completer';
+import { Autocompleter, Completion } from './autocompleter';
+
+export class ShellApiAutocompleter implements Autocompleter {
+  private serverVersion: string;
+
+  constructor(serverVersion) {
+    this.serverVersion = serverVersion;
+  }
+
+  async getCompletions(code: string): Promise<Completion[]> {
```

Comment:
I am not sure I understand why this needs to be async just yet? `cliReplCompleter` does not return anything async just yet, and might not do that for a quite a bit still.

Reply:
We can make it sync, i stared with one autocompleter / without the adapter and ACE had the callback for some reason